src/patterns/filtering/single-selection.stories.ts

```diff
@@ -0,0 +1,142 @@
+
+@Component({
+    selector: "app-sample-single-selection",
+    template: `
+    <div ibmGrid>
+        <div ibmRow class="header">
+            <ibm-header name="Patterns">
+                <ibm-hamburger></ibm-hamburger>
+            </ibm-header>
+        </div>
+        <div ibmRow>
+            <div ibmCol [columnNumbers]="{'lg':3, 'md':3, 'sm':3}">
+                <label ibmText class="dropdown-label">
+                    Filter by:
+                    <ibm-dropdown
+                        class="filter-dropdown"
+                        placeholder="Type"
+                        inline="true"
+                        (selected)="onSelected($event)">
+                        <ibm-dropdown-list [items]="items"></ibm-dropdown-list>
+                    </ibm-dropdown>
+                </label>
+            </div>
+        </div>
+        <div ibmRow>
+            <div ibmCol [columnNumbers]="{'lg': 12, 'md': 12, 'sm': 12}">
+                <ibm-table-container>
+                    <ibm-table
+                    class="data-table"
+                        [model]="model"
+                        size="lg"
+                        [showSelectionColumn]="false">
+                        <ng-content></ng-content>
+                    </ibm-table>
+                </ibm-table-container>
+            </div>
+        </div>
+    </div>
+    `,
+    styles: [`
+        .header {
+            margin-bottom: 80px;
+        }
+
+        .dropdown-label {
+            display: flex;
+            align-items: center;
+            flex-direction-row;
+        }
+
+        .data-table {
+            width: 100%;
+        }
+        
+        .filter-dropdown {
+            flex-grow: 1;
+            margin-left: 20px;
+        }
+
+    `
+    ]
+})
+class SampleSingleSelection implements OnInit {
+    model = new TableModel();
+
+    selected = [];
+
+    items = [
+        { content: "All" },
+		{ content: "Vegetable" },
+        { content: "Fruit" },
+        { content: "Meat" }
+    ];
+
+    dataset = [
+        { name: "Apple", type: "Fruit" },
+        { name: "Grape", type: "Fruit" },
+        { name: "Eggplant", type: "Fruit" },
+        { name: "Lettuce", type: "Vegetable" },
+        { name: "Daikon Radish", type: "Vegetable" },
+        { name: "Beef", type: "Meat" }
+    ];
+    
+    onSelected(event) {
+        this.model.data =
+            this.dataset
+                .filter(data => data.type === event.item.content || event.item.content === "All")
+                .map(filteredData =>                     
+                    [
+                        new TableItem({ data: filteredData.name }),
+                        new TableItem({ data: "Lorem ipsum dolor sit amet, consectetur adipiscing elit." })
+                    ]);
+    }
+
+    ngOnInit() {
+        document.querySelector('.sb-show-main').classList.add('full-page');
+
+        this.model.header = [
+            new TableHeaderItem({
```

Comment:
There's very little reason for these not to be on one line, esp since every other ones are :)